fe/app/components/MentorEventBar/MentorEventBar.tsx

```diff
@@ -0,0 +1,44 @@
+import ImageElement from "@components/ImageElement/ImageElement";
+import { MentorEventBarProps } from "@components/MentorEventBar/MentorEventBar.models";
+import MentorIdentifier from "@components/MentorIdentifier/MentorIdentifier";
+import RoundArrowButton from "@components/RoundArrowButton/RoundArrowButton";
+import { categoriesDictionary } from "@utils/categoriesDictionary";
+
+const MentorEventBar = ({
+  category,
+  title,
+  date,
+  mentor,
+  onClick,
+  past = false,
+}: MentorEventBarProps) => {
+  const _date = {
+    day: date.toString().split(" ")[2],
+    month: date.toString().split(" ")[1],
+  };
+
+  return (
+    <div
+      className={`h-[52px] px-3 py-2 gap-x-2 md:gap-x-40 md:h-[92px] md:py-[14px] md:px-8 ${
+        past ? "border md:border-[2px]" : "bg-" + category
```

Comment:
Here it makes sense to use clsx, so that it's clearer what happens in the case of true and false. I don't love the name "past", I suggest replacing it with "disabled"

Reply:
But is it completely disable? Or is it possible to visit that past event?

Reply:
I could change to "previous"

Reply:
@mariasbl 
- [ ] past
- [ ] disabled
- [ ] previous

@danielaprado 
- [ ] past
- [ ] disabled
- [ ] previous

@catarinapm 
- [ ] past
- [ ] disabled
- [ ] previous

Reply:
Changed to "previous"